ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Deployment/DeploymentPageDetails.tsx

```diff
@@ -0,0 +1,217 @@
+import React from 'react';
+import {
+    Bullseye,
+    Card,
+    DescriptionList,
+    DescriptionListDescription,
+    DescriptionListGroup,
+    DescriptionListTerm,
+    Divider,
+    Flex,
+    FlexItem,
+    Label,
+    LabelGroup,
+    PageSection,
+    Spinner,
+    Text,
+} from '@patternfly/react-core';
+import { gql, useQuery } from '@apollo/client';
+
+import TableErrorComponent from 'Components/PatternFly/TableErrorComponent';
+import { getDateTime } from 'utils/dateUtils';
+
+export type DeploymentPageDetailsProps = {
+    deploymentId: string;
+};
+
+type DeploymentDetails = {
+    id: string;
+    name: string;
+    cluster: {
+        id: string;
+        name: string;
+    };
+    namespace: string;
+    replicas: number;
+    created: string;
```

Comment:
Some of these fields can possibly be null, so I'd be in favor of adjusting the types here to avoid a possible runtime error.
```suggestion
    cluster: {
        id: string;
        name: string;
    } | null;
    namespace: string;
    replicas: number;
    created: string | null;
```